pandas/tests/io/test_feather.py

```diff
@@ -74,6 +77,18 @@ def test_stringify_columns(self):
         df = pd.DataFrame(np.arange(12).reshape(4, 3)).copy()
         self.check_error_on_write(df, ValueError)
 
+    def test_read_columns(self):
+
+        df = pd.DataFrame({'col1': list('abc'),
+                           'col2': list(range(1, 4)),
+                           'col3': list('xyz'),
+                           'col4': list(range(4, 7))})
+        self.check_round_trip(df, columns=None)
+        self.check_round_trip(df, columns=df.columns)
+        random_cols = np.random.choice(df.columns, 2)
```

Comment:
This could select a column twice. I'm not sure what feather does in that case, so I don't know how to read this test.

If feather does allow duplicate `columns` and returns a dataframe with duplicate columns, we could explicitly test that.

If not, then let's just avoid that case by specifying `columns=['col1', 'col2']`.

Reply:
@TomAugspurger I missed `replace=Flase` argument that will ensure we are getting unique columns, this should work
```
        random_cols = np.random.choice(df.columns, 2, replace=False)
```